### PR TITLE
add advancement cargo input node and add choice slot for furnace

### PR DESCRIPTION
**Pull request by Dinnerwolph**
add advanced cargo input node 
add choice slot for furnace
fix https://github.com/TheBusyBiscuit/Slimefun4/issues/114

### CODE REVIEW

File: src/me/mrCookieSlime/Slimefun/Lists/SlimefunItems.java [hunk 1]
```diff
@@ -704,7 +705,7 @@
...
             PROGRAMMABLE_ANDROID_FARMER = new CustomItem(CustomSkull.getItem("eyJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvZjlkMzMzNTdlODQxODgyM2JmNzgzZGU5MmRlODAyOTFiNGViZDM5MmFlYzg3MDY2OThlMDY4OTZkNDk4ZjYifX19"), "&cProgrammable Android &7(Farmer)", "", "&8\u21E8 &7Function: Farming", "&8\u21E8 &7Fuel Efficiency: 1.0x");
-			PROGRAMMABLE_ANDROID_MINER = new CustomItem(CustomSkull.getItem("eyJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvZTYzOGEyODU0MWFiM2FlMGE3MjNkNTU3ODczOGUwODc1ODM4OGVjNGMzMzI0N2JkNGNhMTM0ODJhZWYzMzQifX19"), "�cProgrammable Android &7(Miner)", "", "�8\u21E8 �7Function: Mining", "�8\u21E8 �7Fuel Efficiency: 1.0x");
+			PROGRAMMABLE_ANDROID_MINER = new CustomItem(CustomSkull.getItem("eyJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvZTYzOGEyODU0MWFiM2FlMGE3MjNkNTU3ODczOGUwODc1ODM4OGVjNGMzMzI0N2JkNGNhMTM0ODJhZWYzMzQifX19"), "�cProgrammable Android &7(Miner)", "", "�8\u21E8 �7Function: Mining", "�8\u21E8 �7Fuel Efficiency: 1.0x");
```
Comment (andris155) [inline src/me/mrCookieSlime/Slimefun/Lists/SlimefunItems.java]: �?

Comment (Dinnerwolph) [reply src/me/mrCookieSlime/Slimefun/Lists/SlimefunItems.java]: it was § and intellij had not the right format, but that is correct in the second push

**Comment by Poslovitch** [comment]
That's a shame it has conflicts :(
It was a good quality PR. You should repull it to make sure it won't have any conflicts.

**Comment by Poslovitch** [comment]
Nope.

When a PR has conflicts, you have to close it, redo everything on a new branch, and then PR.
What you did is you just removed a machine added in the latest release.

**Pull request closed without merging**